src/app/shared/directives/course-plate.directive.ts

```diff
@@ -0,0 +1,28 @@
+import { Directive, ElementRef, HostListener, Input, OnInit, Renderer2 } from '@angular/core';
+
+@Directive({
+  selector: '[coursePlate]'
+})
+export class CoursePlateDirective implements OnInit{
```

Comment:
create a separate module for directives and pipes. The same you did for compoenents

Reply:
done